src/Database/Expression/QueryExpression.php

```diff
@@ -532,6 +532,20 @@ public function equalFields($left, $right)
         return $this->eq($wrapIdentifier($left), $wrapIdentifier($right));
     }
 
+    /**
+     * Adds a new type convertion to the expression object in the form "CAST(field AS type)".
+     *
+     * @param string $field Database field to be cast against type
+     * @param string $type target data type to cast field to it
+     * @return $this
+     */
+    public function convert($field, $type)
```

Comment:
Why not name the function `cast()` to match the SQL function name?

Reply:
the issue request convert name. because of portability I used the  cast instead of convert.